### DISCUSSION TITLE
How to display the content in a dedicated component without using the editor?

### DISCUSSION THREAD

**Discussion by GitHubish** [ueberdosis/tiptap#1406] [General Questions] [answered]
We use Tiptap and store the content on the server side in JSON. Only, we need to display tickets on an dashboard (there are hundreds of them). If I use Tiptap (in read only mode) the time to render is way too long. Is there a way to interpret the content to generate HTML for example ? I was looking at prosemirror-view but I don't think it's possible. How to solve this problem?

**Comment by philippkuehn** [comment, marked as answer]
There are some ways to improve this:

- use a pagination
- use a virtual list (only render items that are visible) 
- store HTML content in your DB
- store JSON and convert it to HTML (in browser or server side https://www.tiptap.dev/api/utilities/html/)
